app/models/changeset/commit.rb

```diff
@@ -28,7 +28,13 @@ def summary
   end
 
   def summary_long
-    @data.commit.message.split("\n").first
+    summary = @data.commit.message.split("\n").first
+
+    if summary.nil?
+      return ""
+    end
+
+    summary
```

Comment:
```suggestion
    @data.commit.message.split("\n").first || ""
```

Reply:
This has been updated to reflect your one liner.